fiftyone/operators/types.py

```diff
@@ -2333,6 +2355,44 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
 
+class ModalView(Button):
```

Comment:
@lanzhenw TextView changes are lost in here during a merge conflict resolution. can you include the changes in https://github.com/voxel51/fiftyone/pull/4964 -

Reply:
Yes, just added it.